generic/stress-ng.py

```diff
@@ -198,11 +198,12 @@ def test(self):
                       "\n".join(ERROR))
 
     def tearDown(self):
-        if 'filesystem' in self.class_type:
+        if hasattr(self, 'loop_dev') and os.path.exists(self.loop_dev):
             process.run("umount %s" % self.loop_dev, ignore_status=True,
                         sudo=True)
             process.run("losetup -d %s" % self.loop_dev, ignore_status=True,
                         sudo=True)
+        if os.path.exists('/tmp/blockfile'):
             process.run("rm -rf /tmp/blockfile", ignore_status=True, sudo=True)
-            if (os.path.exists(self.stressmnt)):
-                process.run(f"rm -rf {self.stressmnt}")
+        if hasattr(self, 'stressmnt') and os.path.exists(self.stressmnt):
```

Comment:
@disgoel is `stressmnt` and `/tmp/blockfile` are the same?
if so why are we using variable at one place and hardcoded value at another place?
prefer not to hard code as it may fail if variable name differs

Reply:
both are different
self.stressmnt is a directory and /tmp/blockfile is a block device